ScoutSuite/providers/gcp/facade/stackdriverlogging.py

```diff
@@ -0,0 +1,9 @@
+# -*- coding: utf-8 -*-
+
+from google.cloud import logging as stackdriver_logging
+
+class StackdriverLoggingFacade:
+    # TODO: Make truly async
+    async def get_sinks(self, project_id):
+        client = stackdriver_logging.Client(project=project_id)
+        return client.list_sinks()
```

Comment:
This method returns an iterator. From my understanding, this means the heavy processing will be done by the object iterating over the returned iterator. It might be a good idea to iterate over the sinks, put them in a list and return that list. This way, parallelization can be handled here, and the consumer won't have a bad surprise.  See the [documentation](https://googleapis.github.io/google-cloud-python/latest/_modules/google/cloud/logging/client.html).

Reply:
Good catch!